src/style-sheets/generic/_button.scss

```diff
@@ -1,32 +1,42 @@
 .button {
-    width: 189px;
-    height: 49px;
-    border-radius: 4px;
-    border-width: 0;
+  width: 189px;
+  height: 49px;
+  border-radius: 4px;
+  border-width: 0;
 
-    font-size: 1.25rem;
-    cursor: pointer;
+  font-size: 1.25rem;
+  cursor: pointer;
 }
 
 .primary {
-    background-color: $olLTAccent;
-    color: $white;
-    font-family: $olBaseFont;
+  background-color: $olLTAccent;
```

Comment:
using sass we try and nest common rules. These selectors can be nested as shown:

```
.primary {
  ...
  
  &:hover {
     ...
  }

  &:active {
    ....
  }
}